### ISSUE TITLE
Solution fitness may be incorrectly reported 

### ISSUE THREAD

**Issue by asher-k** [VeryEager/transient-terminal-gp#13]
The average fitness of the best solution may be incorrectly reported by the program, skewing the results. This may be an issue in where the fitness of an individual is calculated.

**Comment by asher-k** [comment]
Fitness of solution in logs does not represent actual fitness of solution as the logs are calculated from the test set, _not_ the training set. Merely an observation.

**Comment by asher-k** [comment]
During meeting state of RMSE results were brought up, more similar in appearance to MSE. Sklearn has function _mean_squared_error_ with Boolean to determine whether to square, though this may still return the MSE?

**Comment by asher-k** [comment]
Further investigation suggests there is no issue with scikit-learn's implementation of MSE, and manually calculating the RMSE returns the same result as passing _False_ to the _mean_squared_error_ function
